### DISCUSSION TITLE
4.6.9

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#230] [Announcements]
#### CHANGED 
- 支持长评论折叠及设置折叠评论长度
  - 感谢 @GD-Slime 进行开发 #214
- 隐藏没有图片的评论的图片滚动条
  - 感谢 @GD-Slime 进行开发 #214
- 用户投稿切换gRPC实现，修复番剧出差用户投稿视频无法打开
  - ⚠ 不再支持用户投稿分区筛选、同时排序与搜索、最多收藏排序
- 变更Cookie存储方案
#### FIXED
- 修复部分AI字幕无法获取
- 移除已失效的代理服务、修复部分视频无法打开
- 修复短信登录问题
  - 感谢 @michael-eddy 提供解决方案 #116

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.9'>4.6.9</a>.</em>

**Comment by guaijk** [comment]
推荐里的视频点进去会跳转到09年的BV1xx411c7mD这个视频，好像播放量越高跳转几率越大
